README.md

```diff
@@ -0,0 +1,91 @@
+# LibrePhotos installation script for Linux
+
+## requirement 
+  - Debian 11 (python 3.9)
+  - postgresql
+  - redis
+
+## Installation
+
+### Debian like distribution
+
+Execute following commande as root
+~~~
+cd /tmp/
+git clone git@github.com:LibrePhotos/librephotos-linux.git
+cd librephotos-linux
+./install-librephotos.sh 
+~~~
+
+Other way can be use download archive
+~~~
+wget https://github.com/librephotos/librephotos-linux/archive/main.zip -O /tmp/main.zip
```

Comment:
This will work after commit. use https://github.com/tomamplius/librephotos-linux/archive/main.zip before